sale_product_pack/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2019 NaN (http://www.nan-tic.com) - Àngel Àlvarez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Sale Product Pack",
+    "version": "14.0.1.0.2",
```

Comment:
```suggestion
    "version": "14.0.1.0.0",
```

Reply:
@dreispt could you just change the version as requested so we can merge it?

Reply:
Done!